.github/workflows/check-secrets-available.yaml

```diff
@@ -22,4 +22,4 @@ jobs:
         env:
           SECRET_TO_CHECK: '${{ secrets.SECRET_TO_CHECK }}'
         if: ${{ env.SECRET_TO_CHECK != '' }}
-        run: echo "::set-output name=has-secrets::true"
+        run: echo "has-secrets=true" >> $GITHUB_OUTPUT
```

Comment:
@aciba90 do you have a reference for this change ? I don't even a deep knowledge on github actions.
Also, @paride, if you can have a look on it as well

Reply:
Sure. One can see these warnings in any executed integration job, for example: https://github.com/canonical/ubuntu-advantage-client/actions/runs/3839211604

This one is:

```
Check secrets / Check secret
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

and the other one:
```
Packaging (jammy)
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout@v2
```

Reply:
Looks good, thanks!